metricbeat/docs/modules/googlecloud.asciidoc

```diff
@@ -0,0 +1,72 @@
+////
+This file is generated! See scripts/mage/docs_collector.go
+////
+
+[[metricbeat-module-googlecloud]]
+[role="xpack"]
+== Google Cloud Platform module
+
+beta[]
+
+This module periodically fetches monitoring metrics from AWS Stackdriver using
+https://cloud.google.com/monitoring/api/metrics_gcp[Stackdriver Monitoring API] for Google Cloud Platform services.
+Note: extra GCP charges on Stackdriver Monitoring API requests will be generated by this module.
+
+
+[float]
+=== Example configuration
+
+The Google Cloud Platform module supports the standard configuration options that are described
+in <<configuration-metricbeat>>. Here is an example configuration:
+
+[source,yaml]
+----
+metricbeat.modules:
+- module: googlecloud
+  metricsets:
+    - compute
+    - pubsub
+    - firestore
+    - firebase
+    - storage
+    - loadbalancing
+  zone: "your zone"
+  project_id: "your project id"
+  credentials_file_path: "your JSON credentials file path"
```

Comment:
Would it be possible to add a sample credential file here in the docs so as to be sure that users will know the structure of this file?

Reply:
Very good suggestion indeed.